lib/internal/async_hooks.js

```diff
@@ -248,6 +248,12 @@ function newUid() {
   return ++async_id_fields[kAsyncIdCounter];
 }
 
+function getOrSetAsyncId(object) {
+  if (object.hasOwnProperty(async_id_symbol)) return object[async_id_symbol];
```

Comment:
Super minor nit: can we add a line break before `return`? I find these same line if-return statements really hard to read.